cax/main.py

```diff
@@ -288,10 +292,47 @@ def massive():
         docs = list(collection.find(query,
                                     sort=sort_key,
                                     projection=['start', 'number','name',
-                                                'detector', '_id']))
+                                                'detector', '_id', 'processor']))
 
         for doc in docs:
+            
+            #Run certains actions only on local file.
```

Comment:
i guess you mean: processes only on runs that satisfy the condition below (non-existence of 'correction_versions')